src/libopenrave-core/jsonparser/jsonreader.cpp

```diff
@@ -140,9 +154,12 @@ class JSONReader
     JSONReader(const AttributesList& atts, EnvironmentBasePtr penv, const std::string& defaultSuffix) : _penv(penv), _defaultSuffix(defaultSuffix)
     {
         FOREACHC(itatt, atts) {
-            if (itatt->first == "openravescheme") {
+            if (itatt->first == "openravescheme" || itatt->first == "remotescheme") {
                 std::stringstream ss(itatt->second);
                 _vOpenRAVESchemeAliases = std::vector<std::string>((istream_iterator<std::string>(ss)), istream_iterator<std::string>());
+                if (itatt->first == "remotescheme") {
```

Comment:
@JosephBerman In the design, we never added `remotescheme`. Here everything in `_vOpenRAVESchemeAliases` should be remote if `remoteurl` is given.

Reply:
Removed remotescheme